src/pages/gettingStarted.ts

```diff
@@ -0,0 +1,121 @@
+                <h3> 4. Read then display the temperature.</h3>
+                <span class="codeBox">
+                    <pre>while True:</pre>
+                    <pre>    temp = temperature()</pre>
+                    <pre>    display.show(temp)</pre>
+                </span>
+                <h3> And much more! These links have more tutorials:</h3>
+                <h3>
+                    <a href="https://microbit.org/projects/make-it-code-it/">Microbit Tutorials</a>
+                </h3>
+                <h3>
+                    <a href="https://microbit-micropython.readthedocs.io/">Microbit official documentation</a>
+                </h3>
+            </div>
+            <div id="CPX" class="inv">
+                <h2> Tutorial for CPX </h2>
+                <h3> 1. Import the micro:bit library to use it! (required)</h3>
+                <span class="codeBox">
+                    <pre>from adafruit_circuitplayground import cp</pre>
+                </span>
+                <h3> 2. Turn on the little red LED</h3>
+                <span class="codeBox">
+                    <pre>while True:</pre>
+                    <pre>   cp.red_led = True</pre>
+                </span>
+                <h3> 3. Turn up red LED when button A is clicked</h3>
+                <span class="codeBox">
+                    <pre>while True:</pre>
+                    <pre>   if cp.button_a:</pre>
+                    <pre>       cp.red_led = True</pre>
+                </span>
+                <h3> 4. Light up the first neopixel blue</h3>
+                <span class="codeBox">
+                    <pre>cp.pixels[0] = (0, 0, 255)</pre>
+                </span>
+                <h3> And much more! These links have more tutorials:</h3>
+                <h3>
+                    <a href="https://learn.adafruit.com/circuitpython-made-easy-on-circuit-playground-express/circuit-playground-express-library">
+                    Getting started with CPX and CircuitPython</a>
+                </h3>
+                <h3>
+                    <a href="https://github.com/adafruit/Adafruit_CircuitPython_CircuitPlayground/tree/master/examples">
+                    More example code</a>
+                </h3>
+            </div>
+    
+            <script>
+                document
+                    .getElementById('target')
+                    .addEventListener('change', function () {
+                        'use strict';
+                        var visibleElement = document.querySelector('.visibleElement'),   
+                            target = document.getElementById(this.value);
+                        if (visibleElement !== null) {
+                            visibleElement.className = 'inv';
+                        }
+                        if (target !== null ) {
+                            target.className = 'visibleElement';
+                        }
+                });
+            </script>
+        </body>
+    </html>`;
```

Comment:
Any reason for embedding every thing into this HTML, instead of creating a React Pure Component for the same?

Reply:
The main reason is that if I decide to make a React Pure Component, I would have to transpile it to html and regular JS. This means that I would need to have 2 react projects. Since the scope of this tutorial is rather small, I decided to go with a regular html. Another option would to add it to the current project and pass a parameter to return this component.